scipy/spatial/_spherical_voronoi.py

```diff
@@ -187,25 +173,16 @@ def _calc_vertices_regions(self):
 
         # get Convex Hull
         self._tri = scipy.spatial.ConvexHull(self.points)
-
-        # triangles will have shape: (2N-4, 3, 3)
         triangles = self._tri.points[self._tri.simplices]
 
         # get circumcenters of Convex Hull triangles from facet equations
-        # circumcenters will have shape: (2N-4, 3)
```

Comment:
the comments related to space complexity are still useful in N-dimensions, right? we'd just have to generalize them

Reply:
I couldn't figure out how to generalize Euler's formula - I don't think `n_v = 2n - 4` generalizes.  I can look into this.

Reply:
The generalization is described here:
https://en.wikipedia.org/wiki/Euler_characteristic#Generalizations

Reply:
This relation doesn't generalize to arbitrary dimensions.

Reply:
Since 3D is still likely the primary use case, perhaps still worth retaining with that clarification then

Reply:
I have put the comments back in, noting that they are valid for 3D inputs.